person.rb

```diff
@@ -1,4 +1,4 @@
-class Person
+class Person < Nameable
```

Comment:
- [ ] You are inheriting from Nameable currently. But it seems you forgot to require the namable file which is causing a load error when I tried to test your project.

Error screenshot:
<img width="567" alt="Screen Shot 2022-09-28 at 8 14 27 PM" src="https://user-images.githubusercontent.com/39523312/192880077-40c6b23b-d7c3-405c-8803-206295d4562f.png">

What you should do to solve the problem is just require nameable file:
```ruby
require './nameable.rb'

class Person < Nameable
   # content goes here
end
```

Reply:
Thanks for the keen eye.